src/librustc_borrowck/diagnostics.rs

```diff
@@ -314,6 +314,77 @@ let c = &i; // still ok!
 ```
 "##,
 
+E0501: r##"
+This error indicates that a mutable variable which has been captured in a
```

Comment:
This explanation is good. But a little rewriting would be nice I think (a lot of words are repeated and it makes the reading a bit harder than necessary).